adele-booking/src/main/java/com/microservicesteam/adele/booking/boundary/web/BookingController.java

```diff
@@ -0,0 +1,40 @@
+package com.microservicesteam.adele.booking.boundary.web;
+
+import com.microservicesteam.adele.booking.domain.BookingRequest;
+import com.microservicesteam.adele.booking.domain.BookingResponse;
+import com.microservicesteam.adele.booking.domain.BookingService;
+import com.microservicesteam.adele.ticketmaster.model.Ticket;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+import static java.util.Comparator.comparingInt;
+import static java.util.stream.Collectors.toList;
+
+@RestController
+@RequestMapping("/rs/api")
+public class BookingController {
```

Comment:
OK, I will do the change as far as I understood, let's discuss the rest in person.